src/store/ChartStore.js

```diff
@@ -314,6 +317,15 @@ class ChartStore {
             }
             let yaxisLabelStyle = this.yaxisLabelStyle;
             if (yax.yaxisLabelStyle) yaxisLabelStyle = yax.yaxisLabelStyle;
+
+            // try to place price label in the y-axis
+            width -= 16;
+            x += 24;
```

Comment:
current price label has a triangle on the left hand, these constant number used for that.